src/Pos/Update/Poll/Logic/Apply.hs

```diff
@@ -47,7 +47,7 @@ import           Pos.Update.Poll.Types         (ConfirmedProposalState (..),
                                                 ProposalState (..),
                                                 UndecidedProposalState (..),
                                                 UpsExtra (..), psProposal)
-import           Pos.Util.Util                 (Some (..))
+import           Pos.Util                 (Some (..))
```

Comment:
@pva701 definitely should setup stylish-haskell on save, but in this particular case it's irrelevant, because this change has been done by `sed`, I suppose.